compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/window/Popup.skiko.kt

```diff
@@ -69,17 +69,43 @@ actual class PopupProperties @ExperimentalComposeUiApi constructor(
     actual val focusable: Boolean = false,
     actual val dismissOnBackPress: Boolean = true,
     actual val dismissOnClickOutside: Boolean = true,
-    val clippingEnabled: Boolean = true,
+    actual val clippingEnabled: Boolean = true,
     val usePlatformDefaultWidth: Boolean = false,
```

Comment:
Lets commonize `usePlatformDefaultWidth` for common Popup too

Reply:
It's experimental on both Android and our implementations, so it will require separate experimental constructor in common.
I don't see use cases and need for it in common now. Also, not applying it now won't require extra steps for keeping compatibility in future